Yoast/Yoast-theme-public
# adds black friday sale banner to the theme setting options

fixes https://github.com/Yoast/yoast.com/issues/1868
Also adds the yoastcon banner to keep it synced with the main theme on yoast.com

how to test:

- Checkout the branch
- Go to theme settings
- put the banner on
- Banner theme: select black friday 
- Banner variation: none
- Banner html:

```
<a class="black-friday-container" href="shop" title="Black Friday Deals">
<div class="sale"><span>Everything 30% off</span></div>
</a>
```

Of course, change the URL to what it should be.
Please inform @ingeariens when ready.
